tests/bluetooth/bsim_bt/compile.source

```diff
@@ -0,0 +1,36 @@
+function compile(){
```

Comment:
This is probably a stretch, but if we are moving this as a common function, maybe we should comment what inputs (variables) it expects?